build/Publish.targets

```diff
@@ -73,6 +74,12 @@
         <ManifestArtifactData>ShipInstaller=dotnetcli</ManifestArtifactData>
       </FilesToPublish>
 
+      <SymbolPackagesToPublish Include="$(DependencySymbolsDir)\**\*.symbols.nupkg" />
+
+      <FilesToPublishToTransport Include="%(SymbolPackagesToPublish.FullPath)" >
+        <RelativeBlobPath>symbols/%(SymbolPackagesToPublish.FileName)%(SymbolPackagesToPublish.Extension)</RelativeBlobPath>
```

Comment:
This looks like it will work--if I'm reading it right, the path ends up being `assets/symbols/...` by the time it hits the publish task. But...

The way I was mentioning in email is that you could add the symbol packages to `PackagesToPublishToTransport` (or whatever you end up passing into the publish task). `PushToBlobFeed` [detects symbol packages by filename, filters them out from the "normal" packages, and puts it at the right relative blob path automatically](https://github.com/dotnet/buildtools/blob/2a9486a85cb385042e26ada50da9370f996f110d/src/Microsoft.DotNet.Build.Tasks.Feed/PushToBlobFeed.cs#L95-L100). I'd suggest going that way because it means you don't need to code in the `assets/symbols/` path here.

Reply:
Cool, so I can just remove line 80 and `PushToBlobFeed` will do the right thing.

Reply:
Not quite, the autodetection only works in the `PublishFlatContainer=false` (default) case, the one used to push NuGet packages (and NuGet symbol packages). This itemgroup goes to the task with `PublishFlatContainer=true`, which is for any assets you want but doesn't do detection. (In theory I don't see any reason it *shouldn't* do detection if `RelativeBlobPath` isn't defined, but it just doesn't right now.)

I think the additional change needed is changing `FilesToPublishToTransport` to `PackagesToPublishToTransport`.

Reply:
Oh, I didn't refresh and Nate beat me to it. :+1: